### DISCUSSION TITLE
[Mobile][iOS] The keyboard not showing up after the image is chosen from the device

### DISCUSSION THREAD

**Discussion by renntbenrennt** [singerdmx/flutter-quill#342] [Mobile]
The main problem is as stated in the title.

My issue is about [Mobile], and it's iOS specifically

I have tried running `example` directory successfully before creating an issue here.

And the issue reproduced in both the example from the package and our environment.

For further demonstration, here's a video recording of this case:

https://user-images.githubusercontent.com/13610283/129296896-0e270d11-5c39-43e5-b979-9343ee0a2446.MP4

As you can see, after the image is chosen from the device, the keyboard not showing up anymore no matter how I tap on the input area.

I have tried to use code like this to try to raise the keyboard up but none of the `requestFocus` method works...

```dart

      _quillController.addListener(() {

        if (MediaQuery.of(context).viewInsets.bottom == 0) {
         // method one:
          _editorFocusNode.requestFocus();
         // method two: and this two method I used them separately 
         // but the keyboard just fail to show up
          FocusScope.of(context).requestFocus(_editorFocusNode);
        }

      });

```

**Comment by li3317** [comment]
Hello, we can't seem to run into this issue on our end.

**Reply by renntbenrennt** [reply]
@li3317 are you sure? I use the example provided in the package to produce the issue and record the video.

**Reply by singerdmx** [reply]
Please fix your own local issue

**Reply by renntbenrennt** [reply]
@singerdmx I am literally using the application running successfully from the example which come with the package to record this video.

**Reply by renntbenrennt** [reply]
@singerdmx @li3317 

I forgot to clarify one thing about my iOS version, which is 14 and above... and I have tested the package example on lower version is working just fine... 

I believe this issue is caused by that image_picker, which also list some known issue regarding iOS 14 and above...

Now I have some workaround to this issue but have to make some breaking change to the package, which I will submit it later.

**Reply by singerdmx** [reply]
It seems that you can raise an issue in image picker

**Reply by renntbenrennt** [reply]
@singerdmx yes, I have successfully reproduced the issue without the flutter_quill package and simply just using the `TextField` and image_picker...

So, after the image is chosen, the `TextField` won't be able to make the keyboard show up again...

but since I don't have much time right now, this particular issue for flutter will be posted later...

Still, I managed to find a way to workaround this issue with flutter_quill... I will posted the detail in this thread later.

Right now I would like to ask another thing, and since it's not in the topic we are discussing here I will open another post.
